lib/index.js

```diff
@@ -44,6 +44,11 @@ class ElectronDownloader {
   }
 
   get cache () {
+    const oldCacheDirectory = path.join(os.homedir(), './.electron')
+    if (pathExists.sync(path.join(oldCacheDirectory, this.filename))) {
```

Comment:
Async code is often better but I chose for a sync method in this case. Firstly because the path to the cache is needed for all other functionality. Secondly all other code expects this method to be sync. Lastly this fallback will be removed in the future so rewriting other methods to work in an async way with `get cache` sounds unreasonable.